cmd/root.go

```diff
@@ -23,12 +23,20 @@ import (
 	"ldcli/internal/projects"
 )
 
+type APIClients struct {
+	EnvironmentsClient environments.Client
+	FlagsClient        flags.Client
+	MembersClient      members.Client
+	ProjectsClient     projects.Client
+}
+
 func NewRootCommand(
 	analyticsTracker analytics.Tracker,
-	environmentsClient environments.Client,
-	flagsClient flags.Client,
-	membersClient members.Client,
-	projectsClient projects.Client,
+	clients APIClients,
```

Comment:
There isn't much risk that we won't set all the right values since we only call it in one place and it's in the same file.